src/api/libcellml/units.h

```diff
@@ -271,22 +271,66 @@ class LIBCELLML_EXPORT Units: public ImportedEntity
     void getUnit(size_t index, std::string& reference, std::string &prefix, double &exponent, double &multiplier, double &offset) const;
 
     /**
+     * @brief Get the @c unit attributes for the given reference @p reference.
+     *
+     * Get the attributes for the first @c unit that matches the reference @p reference. If
+     * no attributes are set, default attribute values will be returned.
+     *
+     * @overload
+     *
+     * @param reference The @c std::string reference for the @c unit attributes to get.
+     * @param prefix The prefix for this @c unit. Defaults to empty string.
+     * @param exponent The exponent for this @c unit. Defaults to 1.0.
+     * @param multiplier The multiplier for this @c unit. Defaults to 1.0.
+     * @param offset The offset for this @c unit. Defaults to 0.0.
+     */
+    void getUnit(const std::string &reference, std::string &prefix, double &exponent, double &multiplier, double &offset) const;
```

Comment:
I must confess that I got confused by the description of the function and its name. Why not call it `getUnitAttributes()` rather than `getUnit()`? (Same below.)

Otherwise, I know that we haven't agreed on a maximum length for a line, but this one is clearly too for my liking, not least when it comes to reviewing using GitHub. It looks like GitHub considers that a line should be 120 characters long maximum. Maybe we could just do the same? In other words, the above line would actually be written:

```
    void getUnit(const std::string &reference, std::string &prefix, double &exponent, double &multiplier,
                 double &offset) const;
```